packages/web/src/scss/components/UNSTABLE_ProductLogo/_UNSTABLE_ProductLogo.scss

```diff
@@ -0,0 +1,18 @@
+@use 'sass:map';
+@use 'theme' as theme;
+@use '../../tools/breakpoint';
+
+.UNSTABLE_ProductLogo {
+    width: fit-content;
+}
+
+.UNSTABLE_ProductLogo > img,
+svg {
+    width: auto;
+    height: 44px;
+}
+
+.UNSTABLE_ProductLogo--inverted {
+    color: theme.$logo-inverted-color;
```

Comment:
This would works only if the inverted `svg` has `fill="currentColor"`, which currently doesn't work with example where is hardcoded fill color.